packages/cascara/src/modules/DataCheckbox/DataCheckbox.js

```diff
@@ -36,20 +36,18 @@ const DataCheckbox = ({
     attribute && record
       ? getAttributeValueFromRecord(attribute, record)
       : value;
-  const [isChecked, setIsChecked] = useToggle(Boolean(finalValue));
+  const checkbox = useCheckboxState({ state: Boolean(finalValue) });
```

Comment:
@lordpixel I think it is best that we leave out the other work of making these modules more simple and getting the value from the record and passing it in at the Body/Row level in a separate PR. Thank you for leaving that out here so we can get the fix in first. 👍🏽